packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -342,11 +347,27 @@ class WSIViewport extends Viewport implements IWSIViewport {
   };
 
   /**
-   * Need to return this as a function to prevent webpack from munging it.
+   * Encapsulate the dicom microscopy fetch so that it can be replaced
+   * with the browser import function.  Webpack munges this and then throws
+   * exceptions trying to get this working, so this has to be provided externally
+   * as a globalThis.browserImportFunction taking the package name, and a set
+   * of options defining how to get the value out of the package.
    */
-  private getImportPath() {
-    return '/dicom-microscopy-viewer/dicomMicroscopyViewer.min.js';
-  }
+  public static getDicomMicroscopyViewer = async () => {
```

Comment:
@sedghi - see how I'm loading the microscopy viewer here using a window.browserImportFunction to avoid letting webpack mess up the scripts.  I really hate having to do it this way, but webpack just messes things up at so many levels that it is really hard to figure out how to manage it.
If you think it is ok, I will write up a blog about it.